UnitedIncome/components
# [APP-3140] Resets for QuestionAnswer

### Description
> Please provide a brief description of your changes below.

Adds line height resets for the QuestionAnswer component.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Load it up & link it up

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-3140)
